src/reporters/tiny.js

```diff
@@ -0,0 +1,66 @@
+import path from "path";
+
+export default report;
+
+async function report(runningTests, startTime) {
+  console.log("");
+  const summary = await reportSequentially(runningTests);
+  const { failed, passed, endTime } = summary;
+  const ok = color.green(`✓ ${passed}`);
+  const notOk = failed ? color.red(`✗ ${failed}`) : "";
+  const runTime = color.grey(`Ran in ${endTime - startTime}ms`);
```

Comment:
I think most other test runners would report in seconds 😂 I guess this one is so fast ms will do.